splunk-ingestable-s3-bucket/terraform.tfvars

```diff
@@ -0,0 +1,8 @@
+terragrunt = {
```

Comment:
This is needed because it will be used in the devops repo right? 

Ideally modules should be pure terraform modules and not include terragrunt config.

Reply:
Absolutely, the module started in the Devops Repo, but I thought this might be useful for any  Splunk users and moved it here. I'll fix it ASAP.